plugins/module_utils/policy_utils.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
This question applies for every other file that has this change. Why the change from "python3" to "python"?
it also seems you are giving backward compatibility to Python 2 because of the use of:
`from __future__ import`
We have dropped support for Python 2.